app/objects/c_operation.py

```diff
@@ -310,13 +310,13 @@ async def report(self, file_svc, data_svc, output=False):
                                                technique_name=step.ability.technique_name,
                                                technique_id=step.ability.technique_id))
                 if output and step.output:
-                    step_report['output'] = self.decode_bytes(file_svc.read_result_file(step.unique))
+                    results_dict = self.decode_bytes(file_svc.read_result_file(step.unique))
```

Comment:
take out type in variable name - 'results_dict' -> 'results'

(change other instances as well)